### ISSUE TITLE
Need to way to reinstall

### ISSUE THREAD

**Issue by pepa65** [Authenticator-Extension/Authenticator#121]
Forgot the Authenticator-Extension password. No problem, I'll just add my accounts back again with a new password. Except I can't, there is no way to wipe, no way to start afresh. I have tried to remove the extension and add it anew, it still remembers my obsolete storage. Do I have to move to a different extension?

**Comment by mymindstorm** [comment]
Are you on Chrome or Firefox?

Have you tried just deleting all the accounts without typing in the correct password?

**Comment by pepa65** [comment]
This is on Firefox. So, I add the extension, click the icon, it asks for the password, so I enter something (and all the accounts show up as either Encrypted or Invalid), and I click the pencil, and try to click on the red-circle sign, and it asks  if I am sure, so I click yes, and keep doing this. Now I have managed to clear all the accounts! (I thought I tried this before, but I got the idea that it didn't work.)

I go to settings, click on Security, and it asks for a password twice (a new password!) and now I can add new entries!!

Love your extension, it's the best.
